### DISCUSSION TITLE
右侧侧边栏的citation key图标不见了？

### DISCUSSION THREAD

**Discussion by Zhang109147** [retorquere/zotero-better-bibtex#3066] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/a1844811-8946-48ef-9333-33e7a6d07ddd)

**Comment by Zhang109147** [comment]
![image](https://github.com/user-attachments/assets/3e0e805d-d872-4ca1-8df1-34d02671ce77)
就在刚刚更新到6.7.253这个版本后，我的citation key图标就不见了

**Reply by retorquere** [reply]
English please.

**Reply by Zhang109147** [reply]
OK，Just now, after updating to version 6.7.253, the citation key icon on the right sidebar disappeared.

**Reply by retorquere** [reply, marked as answer]
Are you on the latest beta? Then the citation key moved to the item info pane. If not, open an issue.

**Reply by Zhang109147** [reply]
Thank you, I can see the citation key now. Thank you very much.
